features/facebookImport/src/components/listOfDetails/listOfDetails.jsx

```diff
@@ -1,7 +1,5 @@
 import React, { useRef } from "react";
-import i18n from "../../i18n";
```

Comment:
Are any changes to this file supposed to be included?

Reply:
Urg this happened through the rebasing

Reply:
Was a commit I did in main, was mb and I'll get it in release to fix this

Reply:
Gone